### ISSUE TITLE
Learning Rate 

### ISSUE THREAD

**Issue by mfaramarzi** [thtrieu/darktf#1161]
I change learning rate in "tiny-yolo-voc-3c.cfg" but it doesn't change in the terminal when I run the training code. Anyone knows why it's happening? 
Any help is really appreciated!
